secrets-provider/ci/test-values-template.yaml

```diff
@@ -0,0 +1,34 @@
+# Values for secrets-provider Chart. All missing values need to be supplied via environment variable.
```

Comment:
I think it should be part of integration tests.
There's no context in this PR to review this.